pkg/config/config.go

```diff
@@ -361,6 +364,11 @@ func (c *Config) SetConfigFromEnv() error {
 		c.VulnerabilityCountLimit = vc
 	}
 
+	// set environment variable for executor type
+	if executor := os.Getenv(controllerBuildrunExecutorEnvVar); executor != "" {
+		c.BuildrunExecutor = executor
+	}
```

Comment:
nit (not blocking): ideally we should have some validation on this env var value and not accept it at face value.